### PR TITLE
[13.0][IMP] purchase_batch_invoicing: improve wizard tree view

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Simplify tree view to see the main data
imagen

### CODE REVIEW

File: purchase_batch_invoicing/views/purchase_order_view.xml [hunk 1]
```diff
@@ -13,4 +13,24 @@
             </xpath>
         </field>
     </record>
+    <record id="purchase_batch_invoicing_tree_view" model="ir.ui.view">
+        <field name="name">purchase.order.tree (purchase_batch_invoicing)</field>
+        <field name="model">purchase.order</field>
+        <field eval="20" name="priority" />
```
Comment (pedrobaeza) [inline purchase_batch_invoicing/views/purchase_order_view.xml]: Put an even higher priority (`999` or so)

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Don't bump module version manually. Bot will do it. I'm wondering if putting the widget `many2many_tags` would be enough.

**Comment by ChristianSantamaria** [comment]
Yes, adding the many2many_tags widget to it would be an option but I think it's fine to see this data

**Review by pedrobaeza — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**